python/modules/IcePy/Util.cpp

```diff
@@ -269,226 +269,132 @@ IcePy::PyObjectHandle::release()
     return result;
 }
 
-IcePy::PyException::PyException()
-{
-    ex = PyErr_GetRaisedException();
-    if (ex)
-    {
-        PyObject* type = reinterpret_cast<PyObject*>(Py_TYPE(ex.get()));
-        Py_INCREF(type);
-        _type = type;
-        _tb = PyException_GetTraceback(ex.get());
-    }
-}
+IcePy::PyException::PyException() { ex = PyErr_GetRaisedException(); }
 
 IcePy::PyException::PyException(PyObject* raisedException)
 {
     Py_XINCREF(raisedException);
     ex = raisedException;
-    PyObject* type = reinterpret_cast<PyObject*>(Py_TYPE(raisedException));
-    Py_INCREF(type);
-    _type = type;
-    _tb = PyException_GetTraceback(raisedException);
 }
 
-void
-IcePy::PyException::raise()
+namespace IcePy
```

Comment:
These should use the same style as the rest:
```
string
IcePy::getTypeName(...
```

Reply:
I move them to the .cpp file because they are local to this file. Updated the code to use tne anonymous namespace instead of IcePy to make it clear.